test-framework/sudo-test/src/docker.rs

```diff
@@ -167,16 +167,41 @@ pub fn build_base_image() {
 
     match SudoUnderTest::from_env() {
         SudoUnderTest::Ours => {
+            #[allow(clippy::vec_init_then_push)]
+            let sudo_build_features: String =
+                env::var("SUDO_BUILD_FEATURES").unwrap_or_else(|_| {
+                    let mut features = vec![];
+
+                    #[cfg(target_os = "freebsd")]
+                    features.push("dev");
```

Comment:
I think we just forgot to remove this for FreeBSD too when disabling it by default for the test framework.

Reply:
I've removed this in the suggestion below.